src/main/resources/de/unijena/cheminf/mortar/message/Message_en_GB.properties

```diff
@@ -364,4 +364,14 @@ ScaffoldGeneratorFragmenter.smilesGeneratorSetting.tooltip = Defines which SMILE
 ScaffoldGeneratorFragmenter.ruleSevenAppliedSetting.tooltip = Defines whether rule 7 of the Schuffenhauer Scaffold Tree dissection method should be applied
 ScaffoldGeneratorFragmenter.retainOnlyHybridisationsAtAromaticBondsSetting.tooltip = Defines whether certain atom hybridisations should be preserved by the insertion of double bonds only at the removal of aromatic rings or always
 ScaffoldGeneratorFragmenter.fragmentationTypeSetting.tooltip = Defines whether only the direct scaffolds should be created or whether they should also be dissected into their parent scaffolds and which method to use for this dissection
-ScaffoldGeneratorFragmenter.sideChainSetting.tooltip = Defines which fragments should be returned, scaffolds, side chains, or both
\ No newline at end of file
+ScaffoldGeneratorFragmenter.sideChainSetting.tooltip = Defines which fragments should be returned, scaffolds, side chains, or both
+##AlkylStructureFragmenter##
+AlkylStructureFragmenter.fragmentSaturationSetting.tooltip = Defines fragment saturation with implicit hydrogen
```

Comment:
This should be the same tooltip text as for all other fragment saturation settings.